kuma/users/jinja2/users/user_edit.html

```diff
@@ -121,14 +130,13 @@ <h2>{{ _('Subscriptions') }} <a class="subscription-link" href="{{ url('payment_
       <ul class="button-container">
         <li>
           <button type="submit" class="positive">
-            {% include 'includes/icons/general/check-mark.svg' %}
-            {{ _('Publish') }}
+            {{ _('Save') }}
           </button>
         </li>
         <li>
           {% if is_me %}
-            <a id="delete-user-button"
-              href="{{ url('users.user_delete', username=request.user.username) }}">{{ _('Delete your account') }}</a>
+            <a id="delete-user-button" class="negative"
+              href="{{ url('users.user_delete', username=request.user.username) }}">{{ _('Delete account') }}</a>
```

Comment:
Personal preference: I liked `Delete your account` better.

Reply:
Je suis ambivalent. But the designs commanded it off me, so I'm going with it.